models/dex/dex_trades.sql

```diff
@@ -78,6 +78,7 @@
 ,ref('woofi_optimism_trades')
 ,ref('oneinch_lop_own_trades')
 ,ref('mummy_finance_optimism_trades')
+,ref('firebird_finance_optimism_trades')
```

Comment:
Do you think we can put `firebird_finance_optimism_trades` into `dex_aggregator.trades`?